core/src/main/java/bio/terra/pearl/core/dao/BaseJdbiDao.java

```diff
@@ -295,6 +295,23 @@ protected List<T> findAllByPropertySorted(String columnName, Object columnValue,
         );
     }
 
+    protected List<T> findAllByTwoPropertiesSorted(String column1Name, Object column1Value,
```

Comment:
With the recent addition of all of these methods like `findXByNProperties[Sorted|Unsorted](...)` etc, should we instead add a method like: `findAllByProperties(Map<String, Object> columns, Optional<SortOptions> sortOptions)`?

We could get rid of `findAllByPropertySorted`, `findAllByTwoPropertiesSorted`, `findAllByProperty`, and `findAllByTwoProperties`, plus avoid the inevitable 3 or 4 property variants, for the small cost of a slightly more complex DAO method that generates the correct SQL.

Reply:
Follow-up ticket works for me 👍